pallets/registrar/src/lib.rs

```diff
@@ -1168,6 +1208,15 @@ pub mod pallet {
                         for para_id in new_paras {
                             Self::cleanup_deregistered_para_id(*para_id);
                             removed_para_ids.insert(*para_id);
+                            if let Err(id) =
+                                BufferedParasToDeregister::<T>::try_mutate(|v| v.try_push(*para_id))
+                            {
+                                log::error!(
+                                    target: LOG_TARGET,
+                                    "Failed to add paraId {:?} to deregistration list",
```

Comment:
This one is ok, we cannot do anything if this fails.

Reply:
_disregard_